aa_stripe/models.py

```diff
@@ -131,7 +131,11 @@ class StripeCoupon(StripeBasicModel):
     def __str__(self):
         return self.coupon_id
 
-    def save(self, *args, **kwargs):
+    def save(self, force_retrieve=False, *args, **kwargs):
+        """Save StripeCoupon
+        Use the force_retrieve parameter to create a new StripeCoupon object form an existing coupon created at
```

Comment:
well that's not really true. 
Force retrieve will update existing coupon too, no?